src/engine/Sort.cpp

```diff
@@ -52,6 +52,19 @@ void Sort::computeResult(ResultTable* result) {
   LOG(DEBUG) << "Getting sub-result for Sort result computation..." << endl;
   shared_ptr<const ResultTable> subRes = _subtree->getResult();
 
+  // TODO<joka921> proper timeout and better estimate
```

Comment:
This is a good question. It is a maintainability issue to probably merge them.
The one is used, when the user requests an ORDER BY, and the other is used, when a SORT is required for technical reasons because an operation like JOIN requires a sorted input. But I assume that this distinction has no technical implication.